### PR TITLE
Adds activeClass property

### LINKED ISSUE #22: Add activeClass prop
We've switched over to using this module from `react-sticky` as we found that unreliable. However, one key feature missing for us here is toggling a class on the element when the sticky is active.

### PR DESCRIPTION
**Pull request by fender**
Resolves #22

### CODE REVIEW

File: src/Sticky.jsx [hunk 1]
```diff
@@ -312,7 +312,7 @@ class Sticky extends Component {
         }
 
         return (
-            <div ref='outer' className={classNames('sticky-outer-wrapper', self.props.className)} style={outerStyle}>
+            <div ref='outer' className={classNames('sticky-outer-wrapper', self.props.className, {[self.props.activeClass]: self.state.status === STATUS_FIXED})} style={outerStyle}>
```
Comment (hankhsiao) [inline src/Sticky.jsx]: It is my first time to see the usage 
```js
{ [someVariable]: aValue }
```
which is pretty cool. My only concern is if it is cross-browser usage? Or actually you can use simpler one like
```js
classNames('sticky-outer-wrapper', self.state.status === STATUS_FIXED ? self.props.activeClass : '')
```

**Comment by hankhsiao** [comment]
Do you mind to add one test for it? I know that's painful part.

**Comment by yahoocla** [comment]
CLA is valid!

File: src/Sticky.jsx (hunk 1 above)
Comment (fender) [inline src/Sticky.jsx]: @hankhsiao It should be fine, and you're using babel on this project anyway right?

Comment (harlowja) [inline src/Sticky.jsx]: This is an automatically generated message.

harlowja@yahoo-inc.com is no longer with Yahoo! Inc.

Your message will not be forwarded.

If you have a sales inquiry, please email yahoosales@yahoo-inc.com and someone will follow up with you shortly.

If you require assistance with a legal matter, please send a message to legal-notices@yahoo-inc.com

Thank you!

**Comment by fender** [comment]
@hankhsiao Added some tests to ensure the class is set as expected.

**Comment by hankhsiao** [comment]
:+1:

**Comment by fender** [comment]
@hankhsiao Thanks! Could we get a bumped version out and on npm?

**Comment by hankhsiao** [comment]
Just did :smile:

**Pull request merged**